packages/block-library/src/more/edit.js

```diff
@@ -54,14 +40,22 @@ export default function MoreEdit( {
 				</PanelBody>
 			</InspectorControls>
 			<div { ...useBlockProps() }>
-				<input
-					aria-label={ __( '“Read more” link text' ) }
-					type="text"
-					value={ customText }
+				<RichText
+					tagName="a"
+					aria-label={ __( '“more” link text' ) }
+					value={ customText || DEFAULT_TEXT }
 					placeholder={ DEFAULT_TEXT }
-					onChange={ onChangeInput }
-					onKeyDown={ onKeyDown }
-					style={ style }
+					onChange={ ( value ) =>
+						setAttributes( { customText: value } )
+					}
+					allowedFormats={ [] }
```

Comment:
If we are disabling all formats, it's better to use PlainText v2 (we should find something better than the v2 at some point)

Reply:
Does it support disableLineBreaks and __unstableOnSplitAtEnd? 🤔

Reply:
OK, I was looking at the Plaintext docs, not EditableText docs  👍